www/README.md

```diff
@@ -1,57 +1,21 @@
-deSEC www
+www
 =====
 
 This docker container provides an nginx-implemented proxy server to all deSEC services. It is the frontend-access method.
 
 
-Requirements
+Certificate Setup
 -----
 
-Although most configuration is contained in this repository, some external dependencies need to be met before the created
-docker image can be run. Dependencies are:
-
-1. Various SSL certificates for the external hostnames. For the following hostnames, put certificate and private key files
-   into a directory and mount it on /etc/ssl/private. Name the files ${NAME}.cer and ${NAME}.key respectively.
-   - `MAIN` (e.g. desec.io)
-   - `www`
-   - `checkip.dedyn`
-   - `checkipv4.dedyn`
-   - `checkipv6.dedyn`
-   - `update.dedyn`
-   - `update6.dedyn`
-   - `dedyn`
-   - `www.dedyn`
-2. DH Parameter file (create this with `openssl dhparam -out dhparam.pem 2048`)
```

Comment:
Is this not needed anymore? I can't find this being done automatically in this commit.

Reply:
This was dead documentation, i.e. the configuration didn't behave that way. Any dhparam.pem file will be ignored (before and after this PR)

Reply:
[This appears to be a non-issue](https://www.ssllabs.com/ssltest/analyze.html?d=desec.io&s=2a01%3a4f8%3a10a%3a1044%3adeec%3a642%3aac10%3a80&latest), as our configuration doesn't support cipher suits relying on dh parameters.